### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by FC5570** [vercel/vercel#6003] [Help] [answered]
I've been trying to host my express.js app for 3 days now, everytime I try to go to my vercel app's url it gives me a 404 error, here is my vercel.json:

`{
    "rewrites": [
        {
            "source": "/(.*)",
            "destination": "/api"
        }
    ]
}`

**Comment by mcsdevv** [comment, marked as answer]
Hi @FC5570, it looks like your rewrite is incorrect, please see this guide: https://vercel.com/guides/using-express-with-vercel
